app/controllers/migration_status_controller.rb

```diff
@@ -52,11 +52,13 @@ def detail
   end
 
   def report
-    filename = "#{params[:id].sub(/:/,'_')}.json"
-    # Sanitize filename for security
-    bar_chars = ['/', '\\']
```

Comment:
We don't need to care about these anymore?

Reply:
I don't think.  I'm pretty sure forward or backslashes would never be in the `id` param.
I manually tested on mco-staging and I was only seeing links with plain old ids e.g. `avalon:12345`.

Reply:
Was the concern that someone malicious could pass a relative path to get the contents of their target?

Reply:
That's probably what it was, but now since I switched it to whitelisting of children of the migration directory I don't think it should be a concern.